packages/client/components/TaskFooterIntegrateMenuList.tsx

```diff
@@ -74,6 +78,18 @@ const TaskFooterIntegrateMenuList = (props: Props) => {
       ... on JiraServerRemoteProject {
         name
       }
+      ... on _xLinearTeam {
+        displayName
+      }
+      ... on _xLinearProject {
```

Comment:
+1 I don't have very good experience with client side resolvers, but I think for this particular use case it could be a good solution by adding a client side `displayName` on this type to avoid `linearTeamAndProjectName` everywhere.

Reply:
I tried implementing a client-side resolver (after first having implemented this as a server-side resolver) and in the ended I decided I didn't like the expression. The best implementation I could come up with was to put a field like `parabolRepoName` on `_xLinearTeam`. Then, if `_xLinearTeam` had a parent of type `_xLinearProject` I could build the `team/project` string. In the end, I felt the resolver was a bit odd as it would implicitly require the query to include the `name` field on `_xLinearProject` to work correctly. In the end I found the utility function a bit more explicit so I went with that.